tests/cases/fourslash/codeFixAddMissingAwait_initializer4.ts

```diff
@@ -15,14 +15,3 @@ verify.codeFix({
   x + y;
 }`
 });
-
-verify.codeFixAll({
```

Comment:
Would it be worthwhile to explicitly confirm that fix-all is unavailable?

Reply:
Added a new fourslash command for it 👍